libs/core/src/lib/grid-list/components/grid-list/grid-list.component.html

```diff
@@ -9,7 +9,7 @@
     [class.fd-grid-list--mode-single-select-right]="selectionMode === 'singleSelectRight'"
     [class.fd-grid-list--mode-multi-select]="selectionMode === 'multiSelect'"
 >
-    <div class="fd-container">
+    <div class="fd-container" role="listbox">
```

Comment:
There are keyboard requirements associated with this role which are not handled in our component. Please check the https://developer.mozilla.org/en-US/docs/Web/Accessibility/ARIA/Roles/listbox_role#keyboard_interactions 

We are missing the "default" focus and the navigation using the arrows.

Reply:
Probably a separate issue can be created for these, since the current structure makes it difficult to add `aria-activedescendant` and `aria-multiselectable`. The current structure does not place the list items under a separate element, and they are siblings with the `title-bar` component; placing `aria-activedescendant` and `aria-multiselectable` on the parent listbox has no effect on the list items. The structure needs to be modified to put the items under another container(which can then serve as a `ul`/listbox).
![image](https://user-images.githubusercontent.com/53509521/121457995-37189e80-c9c6-11eb-8c28-b001fd82ca80.png)

Reply:
I understand but in this case either a new issue needs to be created for addressing the a11y issues for this component or the current PR template should say `Part of #4666` not `Closes #4666`. Because with the current PR we are not adding all required aria roles and keyboard navigation. Please let me know so I know how to proceed with the PR.

Reply:
@InnaAtanasova raised https://github.com/SAP/fundamental-ngx/issues/5655 for the other a11y issues.